yarn.lock

```diff
@@ -10347,23 +10347,23 @@ exponential-backoff@^3.1.1:
   integrity sha512-dX7e/LHVJ6W3DE1MHWi9S1EYzDESENfLrYohG2G++ovZrYOkm4Knwa0mc1cn84xJOR4KEU0WSchhLbd0UklbHw==
 
 express@^4.10.7, express@^4.18.1:
-  version "4.20.0"
-  resolved "https://registry.yarnpkg.com/express/-/express-4.20.0.tgz#f1d08e591fcec770c07be4767af8eb9bcfd67c48"
-  integrity sha512-pLdae7I6QqShF5PnNTCVn4hI91Dx0Grkn2+IAsMTgMIKuQVte2dN9PeGSSAME2FR8anOhVA62QDIUaWVfEXVLw==
+  version "4.21.1"
```

Comment:
Express is used just because `ember-cli`, rather than force express within `yarn.lock` can we look into upgrade `ember-cli` to bring a newer version of express?

Reply:
thats a fair point, @cameronperera opened a PR to upgrade ember-cli. I'll convert this PR to draft and update this PR once the ember-cli upgrade is merged.

Reply:
Sure! Yeah, not saying it will, I did not follow the breadcrumbs what version of express is bringing the newest version of `ember-cli` but in case it does problem solve 😃 otherwise, forcing the version of express is totally fine 😉